src/_mergeServicesAndFactories.js

```diff
@@ -0,0 +1,12 @@
+// TODO: remove this when factories are removed
+export default function mergeServicesAndFactories(options) {
+  const { services = {}, factories = {} } = options;
+  const collidingFactoryNames = Object.keys(factories)
+    .filter(factory => factory in services && factories[factory] !== services[factory]);
+  if (collidingFactoryNames.length) {
+    throw new Error(`Colliding factory names: ${collidingFactoryNames.join(', ')}`);
```

Comment:
At what level is this error is being triggered?

If an app triggers this error, would it block the load of the rest of the apps or only of that specific app?